BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -120,6 +135,46 @@ public class AppProcessor {
         }
     }
 
+    /// Handle incoming URL from iOS AppLinks and redirect it to the correct navigation within the App
+    ///
+    /// - Parameter incomingURL: The URL handled from AppLinks.
+    ///
+    public func handleAppLinks(incomingURL: URL) {
+        guard let sanatizedUrl = URL(string: incomingURL.absoluteString.replacingOccurrences(of: "/#/", with: "/")),
+              let components = NSURLComponents(url: sanatizedUrl, resolvingAgainstBaseURL: true) else {
+            return
+        }
+
+        // Check for specific URL components that you need.
+        guard let path = components.path,
+              let params = components.queryItems,
+              let host = components.host else {
+            services.errorReporter.log(error: AppProcessorError.appLinksInvalidURL)
+            return
+        }
+
+        if path == "/finish-signup" {
```

Comment:
🎨 Change this to a guard:
```suggestion
        guard path == "/finish-signup" else {
            services.errorReporter.log(error: AppProcessorError.appLinksInvalidPath)
            return
        }
```